pocs/observatory.py

```diff
@@ -12,6 +12,7 @@
 
 from pocs.base import PanBase
 from pocs.camera import AbstractCamera
+from pocs.dome.bisque import Dome
```

Comment:
The `bisque` dome is a specific type of dome. We want to use the `AbstractDome` instead.